test/k8sT/Nightly.go

```diff
@@ -406,81 +406,5 @@ var _ = Describe("NightlyExamples", func() {
 				clusterIP, appPods[helpers.App3])
```

Comment:
Please fix the typo in your commit message (minukube --> minikube). Please also reference which GSG in the commit message (i.e., `Documentation/...`) for reference.

Reply:
done